Index: conf/sql.tld
===================================================================
--- conf/sql.tld	(revision 218345)
+++ conf/sql.tld	(revision 218346)
@@ -2,7 +2,7 @@
 
 <taglib xmlns="http://java.sun.com/xml/ns/j2ee"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee web-jsptaglibrary_2_0.xsd"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-jsptaglibrary_2_0.xsd"
     version="2.0">
     
   <description>JSTL 1.1 sql library</description>
Index: conf/c.tld
===================================================================
--- conf/c.tld	(revision 218345)
+++ conf/c.tld	(revision 218346)
@@ -2,7 +2,7 @@
 
 <taglib xmlns="http://java.sun.com/xml/ns/j2ee"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee web-jsptaglibrary_2_0.xsd"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-jsptaglibrary_2_0.xsd"
     version="2.0">
     
   <description>JSTL 1.1 core library</description>
Index: conf/fn.tld
===================================================================
--- conf/fn.tld	(revision 218345)
+++ conf/fn.tld	(revision 218346)
@@ -2,7 +2,7 @@
 
 <taglib xmlns="http://java.sun.com/xml/ns/j2ee"
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee web-jsptaglibrary_2_0.xsd"
+  xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-jsptaglibrary_2_0.xsd"
   version="2.0">
     
   <description>JSTL 1.1 functions library</description>
Index: conf/permittedTaglibs.tld
===================================================================
--- conf/permittedTaglibs.tld	(revision 218345)
+++ conf/permittedTaglibs.tld	(revision 218346)
@@ -2,7 +2,7 @@
 
 <taglib xmlns="http://java.sun.com/xml/ns/j2ee"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee web-jsptaglibrary_2_0.xsd"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-jsptaglibrary_2_0.xsd"
     version="2.0">
   <description>
     Restricts JSP pages to the JSTL tag libraries
Index: conf/fmt.tld
===================================================================
--- conf/fmt.tld	(revision 218345)
+++ conf/fmt.tld	(revision 218346)
@@ -2,7 +2,7 @@
 
 <taglib xmlns="http://java.sun.com/xml/ns/j2ee"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee web-jsptaglibrary_2_0.xsd"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-jsptaglibrary_2_0.xsd"
     version="2.0">
     
   <description>JSTL 1.1 i18n-capable formatting library</description>
Index: conf/scriptfree.tld
===================================================================
--- conf/scriptfree.tld	(revision 218345)
+++ conf/scriptfree.tld	(revision 218346)
@@ -2,7 +2,7 @@
 
 <taglib xmlns="http://java.sun.com/xml/ns/j2ee"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee web-jsptaglibrary_2_0.xsd"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-jsptaglibrary_2_0.xsd"
     version="2.0">
   <description>
     Validates JSP pages to prohibit use of scripting elements.
Index: conf/x.tld
===================================================================
--- conf/x.tld	(revision 218345)
+++ conf/x.tld	(revision 218346)
@@ -2,7 +2,7 @@
 
 <taglib xmlns="http://java.sun.com/xml/ns/j2ee"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee web-jsptaglibrary_2_0.xsd"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-jsptaglibrary_2_0.xsd"
     version="2.0">
     
   <description>JSTL 1.1 XML library</description>

